src/app/visit/visit.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <memory>
+#include <set>
+#include <string>
+
+class Doctor;
+
+class Visit : public std::enable_shared_from_this<Visit>
+{
+    Visit();
+    static std::set<Visit *> visitExtent;
+    std::shared_ptr<Doctor> docAssociation_;
+
+    std::string visitInfo_;
```

Comment:
I think that in the final version we should have specific fields composing visitInfo instead of std::string.
Or it should be of struct type called VisitInfo for example, where we could insert Date, Room, Doctor, Patient, TypeOfVisit etc.).
Storing it, returning or setting  in a simple string might create tons of problems in other code places where we want to extract particular info or set it (with string info validation mainly but perhaps not only).